src/app/app/(application)/minha-conta/page.tsx

```diff
@@ -81,11 +79,11 @@ const MyAccountPage = () => {
         await usersService
           .eraseUser(session?.user.id, values.password)
           .then(() => {
+            setTimeout(() => signOut({ redirect: true }), 1500);
```

Comment:
We need this "setTimeout" function? Because Then is going start after eraseUser resolve the promise.

Reply:
If i not use, the user cant see the toasty